src/manifest.json

```diff
@@ -4,6 +4,7 @@
     "version": "0.3.3",
     "offline_enabled": true,
     "description": "The bookmarking tool that works like your brain: Annotate & Full-Text Search anything you've seen on the web.",
+    "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
```

Comment:
Probably need an explanation for this one too. Can you break this CSP down for me? We actually got kicked from Firefox addons a while back for releasing with a too open CSP, so we want to try and avoid that.

Reply:
This content security policy is to get the `<script>` tags to work in the PDF.js viewer. The script tags are how I am adding the content scripts to the viewer. The 'unsafe-eval' directive allows inline execution of JS with script tags.

Reply:
`unsafe-eval` is the one that got us kicked off last time. I actually found the part in the MDN docs they state that:
http://memex.link/gCB4CjcUO/developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_security_policy

But from what I understand of the docs (and the Chrome-specific docs: https://developer.chrome.com/extensions/contentSecurityPolicy#relaxing-eval), `unsafe-eval` is **not** related to the inline execution of JS in script tags. 

According to the docs, it is possible, although it seems super painful.
Chrome: http://memex.link/yHOqrZVMK/developer.chrome.com/extensions/contentSecurityPolicy#JSExecution
MDN: http://memex.link/zJ1uYQjkq/developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_security_policy

Can you link me to where you are adding the content scripts in the PDF viewer, or doing inline JS, via `<script>` tags?